### DISCUSSION TITLE
enable the log of CP-SAT solver

### DISCUSSION THREAD

**Discussion by DanielLCH** [google/or-tools#3873] [CP-SAT questions]
Hi, Developers

This is my first time using CP-SAT, and I noticed that there is no log, which is unfriendly for large-scale instances. How can I enable the log of the CP-SAT solver? I believe it has something to do with set_solver_specific_parameters, but I'm unsure about its format and parameter names.

Could you give me a demonstration?

Thanks!

P.S. 

my code: (I used `model_builder` because I need to read `.mps` files)

```python
from ortools.linear_solver.python import model_builder as mb

if __name__ == "__main__":
    model = mb.ModelBuilder()

    # Creates the variables.
    num_vals = 3
    x = model.NewIntVar(0, num_vals - 1, 'x')
    y = model.NewIntVar(0, num_vals - 1, 'y')
    z = model.NewIntVar(0, num_vals - 1, 'z')

    # Creates the constraints.
    model.Add(x  -  y == 1)

    # Creates a solver and solves the model.
    solver = mb.ModelSolver("sat")

    # set parameters
    solver.set_solver_specific_parameters("")
    
    # Solve.
    status = solver.solve(model)
```

**Comment by lperron** [comment]
log_search_progress:true
